avaframe/out3Plot/outAna1Plots.py

```diff
@@ -0,0 +1,274 @@
+        avaName = 'FDComparison' + simName
+        outQuickPlot.generatePlot(dataDict, avaName, outDirTest / 'pics', cfg, {'plots': [], 'difference': [], 'stats': [],
+                                  'differenceZoom': []}, crossProfile=False)
+        vSimi = DFAtls.norm(simiDict['vxSimi'], simiDict['vySimi'], simiDict['vzSimi'])
+        vNumerical = DFAtls.norm(fields['Vx'], fields['Vy'], fields['Vz'])
+        dataDict = {'name2': 'solAnalytic', 'name1': simName, 'data2': vSimi, 'data1': vNumerical, 'cellSize': cellSize,
+                    'suffix': 'FV', 'compareType': 'compToRef', 'simName': simName}
+        avaName = 'FVComparison' + simName
+        outQuickPlot.generatePlot(dataDict, avaName, outDirTest / 'pics', cfg, {'plots': [], 'difference': [], 'stats': [],
+                                  'differenceZoom': []}, crossProfile=False)
+
+        # # option for user interaction
+        if cfg['SIMISOL'].getboolean('flagInteraction'):
+            value = input("give time step to plot (float in s):\n")
+            try:
+                value = float(value)
+            except ValueError:
+                value = 'n'
+        else:
+            value = 'n'
+
+
+def plotProfilesSimiSol(ind_time, outputName, relDict, comSol, simiDict, solSimi, axis):
+    """ Plot flow depth and velocity for similarity solution and simulation results
+
+        Parameters
+        -----------
+        ind_time: int
+            time index for simiSol
+        outputName: str
+            outputName
+        relDict: dict
+            dictionary of release area info
+        comSol: dict
+            dictionary of simulation results and info (particles, fields, indices, time step)
+        simiDict: dict
+            dictionary with similiarty solution for h, u, and xCenter at required time step
+        solSimi: dict
+            dictionary with similiarty solution
+        axis: str
+
+    """
+
+    # get info from dem
+    dem = relDict['dem']
+    demOri = relDict['demOri']
+    ncols = dem['header']['ncols']
+    nrows = dem['header']['nrows']
+    xllc = demOri['header']['xllcenter']
+    yllc = demOri['header']['yllcenter']
+    csz = dem['header']['cellsize']
+
+    # com1DFAPy results
+    fields = comSol['fields']
+    x = comSol['x']
+    y = comSol['y']
+    h = comSol['h']
+    v = comSol['v']
+    vx = comSol['vx']
+    vy = comSol['vy']
+    vz = comSol['vz']
+    outDirTest = comSol['outDirTest']
+    indFinal = comSol['indFinal']
+    showPlot = comSol['showPlot']
+    Tsave = comSol['Tsave']
+
+    # similarity solution results
+    vSimi = simiDict['vSimi']
+    vxSimi = simiDict['vxSimi']
+    vySimi = simiDict['vySimi']
+    vzSimi = simiDict['vzSimi']
+    hSimi = simiDict['hSimi']
+    xCenter = simiDict['xCenter']
+    X = relDict['X']
+    Y = relDict['Y']
+
+    fig1, ax1 = plt.subplots(figsize=(2*pU.figW, pU.figH))
+    ax2 = ax1.twinx()
+
+    if axis == 'xaxis':
```

Comment:
I suggest to refactor this (to make it more readable) by preparing the data first and then do the plot. I also suggest to block ax1 and ax2 commands seperately